Zend/tests/fibers/gh10437.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+GH-10437 (Segfault on suspending a dead fiber)
+--FILE--
+<?php
+
+set_time_limit(1);
+
+register_shutdown_function(function () {
+    Fiber::getCurrent()->suspend();
```

Comment:
I can't find my previous comment anymore, so it might have gone lost...

IMO, `Fiber::getCurrent()` should return `null` here, as I expect the stack to be unwound and shutdown functions being called from `main` then.

/cc @trowski

Reply:
Yes, `Fiber::getCurrent()` should be returning `null`. Fix in #10443.